### ISSUE TITLE
Browse our collection a11y: Headings on green lables

### ISSUE THREAD

**Issue by josieg** [BCcampus/open-sage#156]
**Describe the bug**
Right now, the green "Accessible" "Faculty reviewed" "Adopted" or "Ancillary Resource" labels are marked as heading 4. This causes a few problems:

- Books with no labels have empty h4 tags.
- It doesn't make sense for these labels to be marked as headings, but if we decide to keep them marked as headings, they should be a heading level under the title of the book they are associated with (so heading 5)

**To Reproduce**
View this page with WAVE: https://helga.bccampus.ca/wp/opentextbooks/2-0-find-open-textbooks/browse-our-collection/

**Expected behavior**
I expect that there should be no empty headings and that heading levels show relationships between information.

**Screenshots**
![Screenshot_2019-03-25 Browse Our Collection – open UAT site(1)](https://user-images.githubusercontent.com/23706762/54935093-dda72080-4edc-11e9-94b9-26de3af93c61.png)

**Comment by alex-418** [comment]
Resolved via https://github.com/BCcampus/open-sage/commit/4fb85fecf0e2c012c120732933d2902fc3ce15c9
